packages/@sanity/form-builder/src/inputs/ImageInput/Asset.js

```diff
@@ -0,0 +1,230 @@
+      })
+  }
+
+  handleDialogClose = () => {
+    this.setState({
+      showDialog: false,
+      isDeleting: false
+    })
+  }
+
+  handleMenuAction = action => {
+    if (action.name === 'delete') {
+      this.handleDeleteAsset(this.props.asset)
+    }
+    if (action.name === 'showRefs') {
+      this.setState({
+        showDialog: true,
+        dialogType: 'showRefs'
+      })
+    }
+  }
+
+  handleDialogAction = action => {
+    if (action.name === 'close') {
+      this.handleDialogClose()
+    }
+
+    if (action.name === 'delete') {
+      this.handleDeleteAsset(this.props.asset)
+    }
+  }
+
+  renderMenuItem = item => {
+    const {color, title, icon} = item
+    const Icon = icon
+    return (
+      <div className={color === 'danger' ? styles.menuItemDanger : styles.menuItem}>
+        {icon && <Icon />}&nbsp;&nbsp;{title}
+      </div>
+    )
+  }
+
+  getDialogActions = dialogType => {
+    const actions = []
+
+    if (dialogType != 'cantDelete') {
+      actions.push({name: 'delete', title: 'Delete', icon: TrashIcon, color: 'danger'})
+    }
+
+    actions.push({
+      name: 'close',
+      title: 'Close'
+    })
+
+    return actions
+  }
+
+  render() {
+    const {asset, onClick, onKeyPress} = this.props
+    const {isDeleting, showDialog, dialogType} = this.state
+    const size = 75
+    const width = get(asset, 'metadata.dimensions.width') || 100
+    const height = get(asset, 'metadata.dimensions.height') || 100
+
+    const menuItems = [
+      {
+        name: 'showRefs',
+        title: 'Show documents using this',
+        icon: LinkIcon
+      },
+      {
+        name: 'delete',
+        title: 'Delete',
+        color: 'danger',
+        icon: TrashIcon
+      }
+    ]
+
+    return (
+      <a
+        className={styles.item}
+        tabIndex={0}
+        data-id={asset._id}
+        onKeyPress={onKeyPress}
+        style={{
+          width: `${(width * size) / height}px`,
+          flexGrow: `${(width * size) / height}`
+        }}
+      >
+        <i className={styles.padder} style={{paddingBottom: `${(height / width) * 100}%`}} />
+        {/* We can not determine an alt text on image */}
+        <img
+          src={`${asset.url}?h=100`}
+          className={styles.image}
+          onClick={onClick}
+          data-id={asset._id}
+        />
+        {isDeleting && (
+          <div className={styles.spinnerContainer}>
+            <Spinner center />
+          </div>
+        )}
+        <div className={styles.menuContainer}>
+          <DropDownButton
+            placement="bottom-end"
+            showArrow={false}
+            items={menuItems}
+            renderItem={this.renderMenuItem}
+            onAction={this.handleMenuAction}
+          >
+            <MoreVertIcon />
+          </DropDownButton>
+          {showDialog && (
+            <Dialog
+              title={
+                dialogType === 'cantDelete'
+                  ? `Can't delete asset. Used in other documents`
```

Comment:
Change copy to `Cannot delete asset because it is referenced in documents`.

Preferably:
Title: `Could not delete asset`
Subtitle: `The asset is referenced in documents`

Reply:
Good idea to be clear in title and explain in subtitle. Will implement. 
However, after a discussion with @evenwestvang we decided not to use the reference term, since it is not a editor-friendly word.

Reply:
OK – what did you decide to use instead?

Reply:
We decided to use "used in documents"